packages/server/src/user/model/user.model.ts

```diff
@@ -0,0 +1,34 @@
+import { ObjectType, Field, ID } from '@nestjs/graphql';
+import JSON from 'graphql-type-json';
+
+/**
+ * This is a model which adds the needed annotations to the Prisma Project
+ * type for GraphQL. This is not what actually defines the database schema
+ * but does need to be updated if the database schema changes.
+ */
+@ObjectType()
+export class UserModel {
+  @Field(() => ID)
+  id: string;
+
+  @Field()
+  projectId: string;
+
+  @Field({ nullable: true })
+  username?: string;
+
+  @Field({ nullable: true })
+  email?: string;
+
+  @Field()
```

Comment:
minor nit: role is an int
```suggestion
  @Field(() => Int)
```